core/src/com/unciv/logic/battle/BattleDamage.kt

```diff
@@ -208,10 +208,23 @@ object BattleDamage {
 
     private fun getTileSpecificModifiers(unit: MapUnitCombatant, tile: TileInfo): HashMap<String,Float> {
         val modifiers = HashMap<String,Float>()
-        if(tile.isFriendlyTerritory(unit.getCivInfo()) && unit.getCivInfo().hasUnique("+15% combat strength for units fighting in friendly territory"))
-            modifiers["Himeji Castle"] = 0.15f
-        if(!tile.isFriendlyTerritory(unit.getCivInfo()) && unit.unit.hasUnique("+20% bonus outside friendly territory"))
-            modifiers["Foreign Land"] = 0.2f
+
+        if(tile.isFriendlyTerritory(unit.getCivInfo())) {
+            if (unit.getCivInfo().hasUnique("+15% combat strength for units fighting in friendly territory")) {
```

Comment:
// deprecated as of 3.11.0 use "+[15]% combat strength for units fighting in friendly territory" instead
and change jsons accordingly

Reply:
![s1](https://user-images.githubusercontent.com/69697985/95132830-10380780-0769-11eb-8b71-c4020ee44f3f.jpg)